atomic_reactor/schemas/config.json

```diff
@@ -651,12 +651,20 @@
             "description": "Do not report any media types other than these",
             "allOf": [{"$ref": "#/definitions/media_types"}]
           },
-          "blacklist_srpms": {
```

Comment:
How are we going to handle this incompatible change? It's both expected datatype change and name change. Bumping major version?

Reply:
well it is incompatible change, but I think I is used mainly just internally
not sure if we need major version because of this

@lcarva ?

Reply:
It's just a name change.

Reply:
It's not just name change, it was list previously, now it's object.